src/renderer/Layout.tsx

```diff
@@ -250,11 +255,18 @@ export default function Layout() {
     const video = state.videoState[category][state.videoIndex];
     const videoFullPath = video.fullPath;
     const key = "videoPanel" + index;
+    const isMythicPlus = video.category === VideoCategory.MythicPlus && video.challengeMode !== undefined
+    let videoPoster = readyPoster;
+
+    // Show a poster of the dungeon for M+ instead of "Select a video"
```

Comment:
That's cool... and maybe is reason for having larger resolutions. Would like to do this for other categories eventually.

Reply:
Yeah, the larger images take up quite a bit of space but I made them large so that they could be scaled down a bit if you wanted that. It looks bad to scale something _up_.

I was thinking about updating the whole `assets/dungeon/`  directory, but since many of the SL dungeons won't be in a rotation for a long time, it seemed moot.